src/json-editor.component.html

```diff
@@ -9,7 +9,7 @@
   </div>
   <div id="editor-middle-column" [ngClass]="{widerColumn6: isPreviewerHidden}" class="col-md-6 middle" [shortcuts]="shortcuts">
     <table id="editor" class="table">
-      <div class="field-wrapper" *ngFor="let key of keys | addAlwaysShowFields:schema | filterAndSortBySchema:schema; trackBy:trackByElement">
+      <div class="field-wrapper" *ngFor="let key of keys | addAlwaysShowFields:schema | filterAndSortBySchema:schema; let first=first; trackBy:trackByElement" [id]="first ? 'first' : ''">
```

Comment:
I will change the name of the `id` because using a class doing the queryselector after bad performance.